src/ReadingBuddy.java

```diff
@@ -434,7 +541,7 @@ public static void createNewBookGroup(SearchPost wantedPost){
         boolean createGroup;
         //create group but do not increment the nextUserBookGroup number until/only increment if okayed to make group
         BookGroup newGroup = new BookGroup(nextUserBookGroup.get(2), wantedPost, currentUser.getUsername());
-        if(currentUser.checkIfInCurrentBooks(wantedPost.getBookID())){
+        if(currentUser.checkIfInCurrentBooks(wantedPost.getBookID()) >= 0){
```

Comment:
changed b/c function now returns the index of the object if found (or -1 if not found) instead of a boolean